test/ngAnimate/animateCssSpec.js

```diff
@@ -28,9 +35,19 @@ describe("ngAnimate $animateCss", function() {
 
   var ss, prefix, triggerAnimationStartFrame;
   beforeEach(module(function() {
-    return function($document, $window, $sniffer, $$rAF, $animate) {
+    return function($document, $sniffer, $$rAF, $animate) {
       prefix = '-' + $sniffer.vendorPrefix.toLowerCase() + '-';
-      ss = createMockStyleSheet($document, $window);
+      ss = createMockStyleSheet($document);
+      ss.addPossiblyPrefixedRule = function(selector, styles) {
```

Comment:
Done in https://github.com/gkalpak/angular.js/commit/a2d5ef7092cc0321ba5458744940bf01edc50a49.